cfgov/jobmanager/tests/models/test_blocks.py

```diff
@@ -10,13 +11,15 @@
 from jobmanager.models.panels import GradePanel, RegionPanel
 
 
-def make_job_listing_page(title, close_date, grades=[], regions=[]):
+def make_job_listing_page(title, close_date=None, grades=[], regions=[],
+                          **kwargs):
     page = mommy.prepare(
         JobListingPage,
         title=title,
-        close_date=close_date,
+        close_date=close_date or timezone.now().date(),
```

Comment:
@vccabral right, like @richaagarwal says it's a useful shorthand to check whether a value for the `close_date` argument was passed in.

Saying this:

```py
close_date or timezone.now().date()
```

is just a shorter way of doing this check:

```py
if close_date is None:
    close_date = timezone.now().date()
```

All non-`None` `datetime` objects are truthy when treated as a boolean value.